web-app/src/services/API/apiResources/types.ts

```diff
@@ -20,7 +20,7 @@ export interface GetUsersResponse {
 }
 
 export interface GetUserPhotoResponse {
-  data: {
+  data?: {
     photo: unknown;
   };
```

Comment:
```suggestion
  data: {
    photo: string | null;
  };
```
What do you think about this?

Reply:
We could avoid non-null operator in other files and casting to string(e.q UserAvatar.vue )

Reply:
This is just leftover. Type null not needed